data/CFOUR/basicCFOUR2.1/cfour.slurm

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
I think this and `cfour.out` were added on accident.

Reply:
Seems that way, both of the files have been removed.